package.json

```diff
@@ -29,9 +29,12 @@
     }
   },
   "devDependencies": {
+    "calc-percent": "1.0.1",
```

Comment:
These are dependencies, not devDependencies as they are part of what's required to run `fastatic`.